docs/transparent-validation/muxpi/theory-of-operations.md

```diff
@@ -33,13 +33,15 @@ RX`, `DBG UART TX` on pinout image below):
 ![](../../images/muxPi-addon-left.png)
 
 and plug USB connector to the computer. Next, open connection in PC terminal by
-typing: `sudo minicom -D /dev/ttyUSB0 -o -b 115200`.
+typing:
+
+    sudo minicom -D /dev/ttyUSB0 -o -b 115200.
 
 > `/dev/ttyUSB0` is example device - check whether `USB0` is the correct one
 
 #### Jumpers configuration
 
-![](../../images/muxPi-jumpers.png)
+![](https://cloud.3mdeb.com/index.php/s/EMpXf8c4JaDNTpH/preview)
```

Comment:
To ensure control over the documentation, we placed all graphics in location `docs/images`. Please put all files into that location and change the linking.